spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsTransmitter.java

```diff
@@ -128,7 +128,7 @@ private void transmitToYggAdmin(eu.coatrack.api.Metric metricToTransmit) {
             */
 
         } catch (Exception e) {
-            log.error("Exception when communicating with CoatRack admin server", e);
+            log.debug("Exception when communicating with CoatRack admin server", e);
```

Comment:
This could be a serious issue in production use, why change it to "debug" level?

Reply:
My thought process was the following: This error occurs when admin is down. In the context of this branch, the gateways expects that admin is sometimes down, even if rarely. That is why I did not want to bloat the log with exceptions caused by normal business logic.